security/aporeto/docs/README.md

```diff
@@ -0,0 +1,337 @@
+```
+
+This should produce output similar to the following:
+
+```console
+NAME                                         AGE
+egress-internet-devex-von-tools              5d
+int-cluster-k8s-api-permit-devex-von-tools   5d
+intra-namespace-comms-devex-von-tools        5d
+```
+
+If you want to dig a little deeper you can specify one of the policies to fetch and output it in well formated YAML. Run the following command to fetch a specific policy:
+
+```console
+oc get networksecuritypolicy egress-internet-devex-von-tools -o yaml
+```
+
+This command produces detailed output similar to the following. You only need to focus on the `spec:` stanza to inspect the rules of the NSP. The other information can safely be ignored.
+
+```yaml
+apiVersion: secops.pathfinder.gov.bc.ca/v1alpha1
+kind: NetworkSecurityPolicy
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"secops.pathfinder.gov.bc.ca/v1alpha1","kind":"NetworkSecurityPolicy","metadata":{"annotations":{},"name":"egress-internet-devex-von-tools","namespace":"devex-von-tools"},"spec":{"description":"allow devex-von-tools to talk to the internet\n","destination":[["ext:network=any"]],"source":[["$namespace=devex-von-tools"]]}}
+  creationTimestamp: "2019-10-02T15:28:48Z"
+  finalizers:
+  - finalizer.secops.pathfinder.gov.bc.ca
+  generation: 1
+  name: egress-internet-devex-von-tools
+  namespace: devex-von-tools
+  resourceVersion: "397972060"
+  selfLink: /apis/secops.pathfinder.gov.bc.ca/v1alpha1/namespaces/devex-von-tools/networksecuritypolicies/egress-internet-devex-von-tools
+  uid: 54615585-e529-11e9-81f0-00505683b471
+spec:
+  description: |
+    allow devex-von-tools to talk to the internet
+  destination:
+  - - ext:network=any
+  source:
+  - - $namespace=devex-von-tools
+status:
+  conditions:
+  - ansibleResult:
+      changed: 3
+      completion: 2019-10-07T16:55:09.793987
+      failures: 0
+      ok: 11
+      skipped: 0
+    lastTransitionTime: "2019-10-02T16:50:52Z"
+    message: Awaiting next reconciliation
+    reason: Successful
+    status: "True"
+    type: Running
+```
+
+**Web - console**
+
+Open the OCP administration console and navigate to the namespace containing the NSP you want to inspect. Next, go to Resources -> Other Resources and use the dropdown to select 'Network Security Policy'.
+
+![Web View NSP](webview_nsp.gif)
+
+### Remove It
+
+To remove network policy you can again use the CLI or Web console. For the CLI just use the `delete` verb in the command you used to inspect policy
+
+```console
+oc delete networksecuritypolicy egress-internet-devex-von-tools -o yaml
+```
+
+From the Web, again navigate to the NSP and use the "Actions" drop down to select "Delete".
+
+### Advanced
+
+## Projects
+
+This is a list of some projects that have already implemented a Zero Trust security model:
+
+[Family Protection Order](https://github.com/bcgov/Family-Protection-Order)
+
+### Troubleshooting
```

Comment:
Add Step 3 - Test it
This section can include a link to the TestConnection script https://github.com/BCDevOps/openshift-developer-tools/blob/master/bin/testConnection and examples on how to use it.

Reply:
Added.